src/frontend/komponenter/Oppgavebenk/DatoPeriode.tsx

```diff
@@ -14,8 +14,8 @@ interface Props {
     datoTilTekst: string;
     valgtDatoFra?: string;
     valgtDatoTil?: string;
-    settDatoFra: (datoFra?: string) => void;
-    settDatoTil: (datoTil?: string) => void;
+    settDatoFra: ((datoFra?: string) => void) | Dispatch<SetStateAction<string>>;
```

Comment:
Jag får TS2322: `Type 'Dispatch<SetStateAction<string>> | ((datoFra?: string | undefined) => void)' is not assignable to type '(date?: string | undefined) => void'.` inne i `DatoPeriode`